### ISSUE TITLE
moved to next generation but the time counter hasn't updated correctly

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#277]
I just moved and now the counter reports a negative timing:

![image](https://cloud.githubusercontent.com/assets/180794/6669115/82ca3be6-cbf7-11e4-9b28-986da791fa02.png)

**Comment by derekpa** [comment]
Doesn't happen with me

**Comment by giovanisp** [comment]
Use chrome. It happens everytime I move a proposal to voting.

**Comment by pietrosperoni** [comment]
Giovani, you are living in the past

**Comment by derekpa** [comment]
I'm kind of at a loss.
![screen shot 2015-05-14 at 10 25 02](https://cloud.githubusercontent.com/assets/1368533/7629341/9dcc0bf4-fa23-11e4-8073-4e0b97d7d49b.png)

@pietrosperoni you get this or are you also living in the past?

**Comment by giovanisp** [comment]
Which browser are you using? Also, i get the "past" message when moving to voting.

Inviato da iPad

**Comment by derekpa** [comment]
Just moved to voting on chrome 40 on macbook. You cleared your cache? Pietro, you getting this bug?

![screen shot 2015-05-15 at 12 34 47](https://cloud.githubusercontent.com/assets/1368533/7652058/a9159386-faff-11e4-8a57-6dc1fae917d8.png)

**Comment by derekpa** [comment]
I wonder if its a timing thing caused by slow loading.

**Comment by derekpa** [comment]
Nope. Can't reproduce this. @giovanisp could you make a video of this?


Pietro does this happen to you- on TEST?

**Comment by giovanisp** [comment]
I will, soon. Anyway seems I am the only one having this issue, so let's postpone as not urgent and just go ahead with 2.0.0.

**Comment by pietrosperoni** [comment]
it is not happening on me
